core/modules/file/file.module

```diff
@@ -1901,6 +1901,7 @@ function file_icon_path(File $file, $icon_directory = NULL) {
 function file_icon_map(File $file) {
   switch ($file->filemime) {
     // Word document types.
+    // cspell:disable
```

Comment:
Rather than several disable/enable blocks around each set of case statements, could we just do one around the whole switch statement? There's very little other text here that isn't being excluded.

Reply:
Done with https://github.com/backdrop/backdrop/pull/4578/commits/982e98c33e8167009a9a70b8be0e793b3f04775d